packages/cli/test/commands/migrate/__snapshots__/e2e.test.ts.snap

```diff
@@ -350,7 +350,7 @@ exports[`migrate: e2e > regen result after the first pass 1`] = `
 `;
 
 exports[`migrate: e2e > user defined options passed by --user-config -u > migrate.exclude 1`] = `
-"[32minfo[39m:    @rehearsal/migrate 1.0.8-beta
+"[32minfo[39m:    @rehearsal/migrate 1.0.9-beta
```

Comment:
I believe that we need to remove the version from the output

Reply:
I changed the test. I assert on the stdout. Will regen the snapshot and it will be gone.